doc/library/config.txt

```diff
@@ -716,6 +716,23 @@ import theano and print the config variable, as in:
 
     Location of the magma library.
 
+.. attribute:: config.ctc.enabled
+
+    Default: ``'False'``
+
+    If ``'True'``, use `warp-ctc <https://github.com/baidu-research/warp-ctc>`_ for CTC
+    loss computations.
+
+    If ``'False'``, disable warp-ctc.
+
+.. attribute:: config.ctc.root
+
+    Default: ``''``
+
+    Location of the warp-ctc folder. The folder should contain a build
+    subfolder with the shared library (libwarpctc.so), and another
```

Comment:
Added support for `lib`, `lib64` and `build` subdirectories within `config.ctc.root`